apps/trackers/common.py

```diff
@@ -366,3 +373,15 @@ def _description_feedback_form(self):
         if TRACKER_FEEDBACK_FORM_URL is None:
             return []
         return [f"Tracker accuracy feedback form: {TRACKER_FEEDBACK_FORM_URL}"]
+
+    def _description_Vuln_Mgmt_info(self):
```

Comment:
I would refrain from using capitalized letters in snake case. I think it is even discouraged in PEP.